config/v1/types_authentication.go

```diff
@@ -85,7 +85,8 @@ type AuthenticationSpec struct {
 	// +listType=map
 	// +listMapKey=name
 	// +kubebuilder:validation:MaxItems=1
-	// +openshift:enable:FeatureSets=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +openshift:enable:ClusterProfileAwareFeatureSets:featureSet=CustomNoUpgrade;TechPreviewNoUpgrade
+	// +openshift:enable:ClusterProfileAwareFeatureSets:featureSet=Default,clusterProfile=ibm-cloud-managed
```

Comment:
Based on observed results, we only see the first instance of the tag, not the second.  Which causes this approach to fail unless we produce a list of the tuples we want.  Which is likely possible.  Not sure we want to.